src/browser-lib/single-spa-angular.ts

```diff
@@ -78,7 +79,13 @@ function mount(opts, props) {
           throw Error(`single-spa-angular: the opts.bootstrapFunction returned a promise that did not resolve with a valid Angular module. Did you call platformBrowser().bootstrapModuleFactory() correctly?`)
         }
         module.injector.get(opts.NgZone)._inner._properties[opts.zoneIdentifier] = true;
-        
+        const ngZone = module.injector.get(opts.NgZone);
+        const ngZoneID = opts.zoneIdentifier;
+        routingListener = () => {
+          //fix browser back / forward button
+          ngZone.run(() => console.log("ngZone for", ngZoneID));
```

Comment:
This would be quite annoying to see in the console all the time. The single-spa-angular community [is already displeased](https://github.com/CanopyTax/single-spa-angular/issues/53) about the "Navigation triggered outside of Angular" warning in the console, and so let's remove the console.log() altogether.